### ISSUE TITLE
How should saved items for other stores be treated?

### ISSUE THREAD

**Issue by foodisready** [foodisready/cooking#24]
#21 

currently it's only showing saved items of the store you're in.  
What to do with the saved items of other stores?
If the current store has no saved items, but there are saved items in other stores, should the save icon be filled?

<img width="391" alt="image" src="https://github.com/user-attachments/assets/2162da30-e240-4329-a365-161f45036966" />

getSavedItems()

**Comment by superfuturefunk** [comment]
No, let's treat store independent from each other. If theres no saved items then don't fill.
